README.md

```diff
@@ -104,6 +105,20 @@ It will not:
 * `['os-hardening']['security']['selinux_mode'] = 'unmanaged'`
   set to `unmanaged` if you want to let selinux configuration as it is. Set to `enforcing` to enforce or `permissive` to permissive SELinux.
 
+### Controlling the included recipes
+
+`default.rb` includes other recipes based on the ohai autodetection attributes of your system. E.g. do not include selinux recipe on non-RHEL systems. You can override this behavior and force recipes to be executed or not via setting attributes in `node['os-hardening']['recipes']` on the override level. Example
+
+```ruby
+# some attribute file
+# do not include sysctl and auditd recipes
+override['os-hardening']['recipes']['sysctl'] = false
```

Comment:
I would call this [os-hardening]['components'], since this frees us from having a 1:1 relationship which we may not want in the future. Except from that this makes perfect sense.